docs/api/covidcast-signals/chng.md

```diff
@@ -62,6 +68,8 @@ upward spikes in the fraction of doctor's visits that are COVID-related around
 major holidays (e.g. Memorial Day, July 4, Labor Day, etc.). These spikes are
 not necessarily indicative of a true increase of COVID-19 in a location.
 
+Note that due to Y, estimates are not always comparable across geo locations. We are currently working on adjustments to correct this spatial bias.
```

Comment:
```suggestion
Note that due to local differences in health coding practices, estimates are not always comparable across locations. We are currently working on adjustments to correct this spatial bias.
```
The intent is that doctors in different locations assign ICD codes differently. I don't know how to express this succintly, since the word "code" usually implies computer code and I don't know how well acquainted our audience is with the term ICD code.

Reply:
"recordkeeping"?